src/inference/src/cpp_interfaces/interface/ie_iinfer_request_internal.cpp

```diff
@@ -95,18 +109,19 @@ void IInferRequestInternal::SetBlob(const std::string& name, const Blob::Ptr& us
     InputInfo::Ptr foundInput;
     DataPtr foundOutput;
     const bool isInput = findInputAndOutputBlobByName(name, foundInput, foundOutput);
```

Comment:
do we need `isInput`?  I suppose now we can use `if (input)`

Reply:
In old API we can have empty input